src/components/NoteDialogComponents/NoteDialogNoteComponents/NoteDialogNoteContent.scss

```diff
@@ -41,14 +41,24 @@
   padding: 0 4px;
   font-size: $text--md;
   line-height: 1.5rem;
-
   &:not(:disabled) {
     &:hover,
     &:focus-visible {
       border-color: var(--accent-color--desaturated-dark);
     }
   }
 }
+.note-dialog__note-content-container {
+  width: 100%;
+  height: 100%;
+}
```

Comment:
don't think this actually makes a difference. if you wanna make any changes to the note please make sure that #4180 isn't impacted by it